tests/test_user_management.py

```diff
@@ -14,7 +14,10 @@ def setUpClass(cls) -> None:
             cls.credentials = json.load(f)
 
     def test_success(self):
-        actual = get_user_information(**self.credentials)
+        url = self.credentials['scheme'] + '://' \
+            + self.credentials['host'] + ':' \
+            + str(self.credentials['port'])
+        actual = get_user_information(url, self.credentials['token'])
 
         # TODO: reduced to required
         expected_keys = [
```

Comment:
The expected keys are all correct, perhaps we can get rid of this TODO?

Reply:
done